build/do.rq

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env rq
+# rq: query data.script.main
+# rq: output null
+import future.keywords
+
+what := rq.args()[0]
+main := do[what] { true }
+else := job[what]
+
+do.pull_request {
+	some x in ["test", "lint", "e2e", "check_readme"]
+	github("::group::", x)
+	job[x]
+	github("::endgroup::", x)
+}
+
+job.pr {
+	print("thanks for contributing")
+}
```

Comment:
This isn't used right?

Reply:
No, I'll add those steps later to help run all steps needed to prepare a PR.